### ISSUE TITLE
first run of kitchen-dokken testing appears to fail

### ISSUE THREAD

**Issue by st-isidore-de-seville** [someara/kitchen-dokken#4]
Hey @someara, I am hooking up kitchen-dokken into some of my private repositories and testing against centos-6 and centos-7.  I am noticing that if I am testing an environment where these images have not been pulled for the first time, I am receiving the following error message:

```
>>>>>> ------Exception-------
>>>>>> Class: Kitchen::ActionFailed
>>>>>> Message: Failed to complete #create action: [No such image: centos
]
>>>>>> ----------------------
>>>>>> Please see .kitchen/logs/kitchen.log for more details
>>>>>> Also try running `kitchen diagnose --all` for configuration
```

However after some time, if I run the command again, it is able to pull the image and I can resume testing.  

Thoughts?

**Comment by someara** [comment]
is it still doing this with the latest version?

**Comment by st-isidore-de-seville** [comment]
@someara let me check and get back with you.

**Comment by st-isidore-de-seville** [comment]
sorry it took me a sec to get back with you but yes this is still happening behind a proxy

**Comment by sworisbreathing** [comment]
@st-isidore-de-seville I'm running it (kitchen-dokken 0.0.23) behind a proxy (local CNTLM listening on port 3129 to the `docker0` adapter) and I'm able to pull the image.

Here's my $HOME/.kitchen/config.yml

```
---
driver:
  http_proxy: http://172.17.42.1:3129
  https_proxy: http://172.17.42.1:3129

provisioner:
  http_proxy: http://172.17.42.1:3129
  https_proxy: http://172.17.42.1:3129
  attributes:
    yum:
      main:
        proxy: http://172.17.42.1:3129
```

and my .kitchen.dokken.yml:

```yml
---
driver:
  name: dokken
  chef_version: 12.5.1
  intermediate_instructions:
    - ENV HTTP_PROXY=http://172.17.42.1:3129
    - ENV http_proxy=http://172.17.42.1:3129
    - ENV HTTPS_PROXY=http://172.17.42.1:3129
    - ENV https_proxy=http://172.17.42.1:3129
    - ENV FTP_PROXY=http://172.17.42.1:3129
    - ENV ftp_proxy=http://172.17.42.1:3129
    - ENV NO_PROXY=<%= ENV['NO_PROXY'] %>
    - ENV no_proxy=<%= ENV['NO_PROXY'] %>
    - RUN echo "proxy=http://172.17.42.1:3129" >> /etc/yum.conf

transport:
  name: dokken

provisioner:
  name: dokken

platforms:
  - name: centos-6.6
    driver:
      image: centos:centos6

verifier:
  root_path: /opt/verifier
  sudo: false
```

**Comment by st-isidore-de-seville** [comment]
@sworisbreathing , hey I will try this again.  is there any debugging flags or commands I can enable that would provide more detailed debugging output?

Thanks!

**Comment by sworisbreathing** [comment]
@st-isidore-de-seville not currently, but I've opened #7 to address this. I'm running it locally and getting tons of output.

**Comment by someara** [comment]
Is this still an issue for you?

**Comment by someara** [comment]
Closing this for now... I'm pretty sure this got fixed somewhere along the road to 1.0.0. Feel free to re-open if this is still an issue.
